Makefile

```diff
@@ -34,13 +40,14 @@ download-ext:
 clean-ext:
 	$(MAKE) -C src_ext distclean
 
-clean: fastclean
```

Comment:
Oops - you've merged before I had a chance to change this... you want to push `clean: fastclean` back...